bin/rpc.ml

```diff
@@ -2,6 +2,7 @@ open! Stdune
 open Import
 
 let wait_for_server common =
```

Comment:
What about `get_server_where` or something?

Reply:
Yes, I'll rename that in other PR's.